npm/src/db/sql/sqlite/entity/JacksonStore.ts

```diff
@@ -0,0 +1,50 @@
+import { Entity, Column } from 'typeorm';
+
+@Entity()
+export class JacksonStore {
+  @Column({
+    primary: true,
+    type: 'varchar',
+    length: 1500,
+  })
+  key!: string;
+
+  @Column({
+    type: 'text',
+  })
+  value!: string;
+
+  @Column({
+    type: 'varchar',
+    length: 64,
+    nullable: true,
+  })
+  iv?: string;
+
+  @Column({
+    type: 'varchar',
+    length: 64,
+    nullable: true,
+  })
+  tag?: string;
+
+  @Column({
+    type: 'datetime',
+    default: () => 'CURRENT_TIMESTAMP',
+    nullable: false,
+  })
+  createdAt?: Date;
+
+  @Column({
+    type: 'datetime',
+    nullable: true,
+  })
+  modifiedAt?: string;
+
+  @Column({
+    type: 'varchar',
+    length: 64,
```

Comment:
This is now 256 in v1.25.0, could we do the same here?

Reply:
We sure can, despite it having [no effect](https://www.sqlite.org/faq.html#q9):

> (9) What is the maximum size of a VARCHAR in SQLite?
> 
> SQLite does not enforce the length of a VARCHAR. You can declare a VARCHAR(10) and SQLite will be happy to store a 500-million character string there. And it will keep all 500-million characters intact. Your content is never truncated. SQLite understands the column type of "VARCHAR(N)" to be the same as "TEXT", regardless of the value of N.